django_cyverse_auth/authBackends.py

```diff
@@ -351,8 +351,9 @@ def keystone_validate_auth(self, auth_url, username, password, project_name, dom
         auth_token = ks_session.get_token()
         expiry_time = password_auth.auth_ref.expires
         try:
+            email = self._lookup_email(session)
```

Comment:
`session` here is `from keystoneauth1 import session` -- I think you want `ks_session`?